connect/src/com/telenor/connect/id/ConnectIdService.java

```diff
@@ -154,4 +155,15 @@ public String getSubjectId() {
 
         return idToken != null ? idToken.getSubject() : null;
     }
+
+    public void getUserInfo(Callback<UserInfo> userInfoCallback)
+            throws ConnectNotInitializedException {
+        String accessToken = getAccessToken();
+        if (accessToken == null) {
+            throw new ConnectNotInitializedException(
```

Comment:
I would actually use a ConnectException, or create a new Exception type for when the user is not signed in.